autometa/common/external/samtools.py

```diff
@@ -61,8 +89,8 @@ def main(args):
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p')
     parser = argparse.ArgumentParser()
```

Comment:
You can place the description of the script when you initialize `argparse.ArgumentParser`. I.e. Perhaps this should be the place where you describe taking a sam file and converting it to a bam file

`parser = argparse.ArgumentParser(description='Description of script to display to user')`